client/post-editor/media-modal/detail/test/index.jsx

```diff
@@ -43,21 +56,19 @@ describe( 'EditorMediaModalDetailItem', () => {
 	const isVideoPressEnabled = jest.fn( () => true );
 
 	test( 'should display at least one edit button for a VideoPress video on a public site', () => {
-		const tree = shallow(
+		renderWithRedux(
 			<DetailItem
 				item={ DUMMY_VIDEO_MEDIA }
 				isVideoPressEnabled={ isVideoPressEnabled }
 				{ ...SHARED_PROPS }
 			/>
 		);
 
-		const editButton = tree.find( '.editor-media-modal-detail__edit' );
-
-		expect( editButton.length ).toBeGreaterThanOrEqual( 1 );
+		expect( screen.queryAllByRole( 'button' ).length ).toBeGreaterThanOrEqual( 1 );
```

Comment:
Would it make sense to be more specific on the buttons we expect? I'm thinking of using something like

```js
screen.queryAllByRole( 'button', { name: /edit/i } );
```

I'm not exactly familiar with the component but if we render other buttons which aren't edit buttons and no edit buttons this assertion will still pass. Haven't tested it, just something which came to mind.

Reply:
Agreed here - the current assertion is too broad, compared to what we had before.

Reply:
Thank you for the suggestion, that makes sense!